### ISSUE TITLE
"TypeError: client.getTorrent is not a function" during mediainfo call

### ISSUE THREAD

**Issue by danmilon** [mpetazzoni/ttorrent#208]
I was inspecting the network requests and found this by accident:
```
GET http://flood-host/api/mediainfo?hash=a-hash

<!DOCTYPE html><html><head><title></title><link rel="apple-touch-icon" sizes="180x180" href="./apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="./favicon-16x16.png"><link rel="manifest" href="./manifest.json"><link rel="mask-icon" href="./safari-pinned-tab.svg" color="#0e2337"><meta name="theme-color" content="#ffffff"><link rel="stylesheet" href="./style.css"><script>window.floodConfig = {baseURI: '', maxHistoryStates: , pollInterval: };</script></head><body><h1>client.getTorrent is not a function</h1><h2></h2><pre>TypeError: client.getTorrent is not a function
    at Object.getMediainfo (/srv/flood/server/util/mediainfo.js:16:12)
    at router.get (/srv/flood/server/routes/api.js:62:13)
    at Layer.handle [as handle_request] (/srv/flood/node_modules/express/lib/router/layer.js:95:5)
    at next (/srv/flood/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/srv/flood/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/srv/flood/node_modules/express/lib/router/layer.js:95:5)
    at /srv/flood/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/srv/flood/node_modules/express/lib/router/index.js:335:12)
    at next (/srv/flood/node_modules/express/lib/router/index.js:275:10)
    at complete (/srv/flood/node_modules/passport/lib/middleware/authenticate.js:250:13)</pre><script src="./app.js"></script></body></html>
```

I'm running the latest master.

**Comment by mpetazzoni** [comment]
I'm confused, this is a Javascript stacktrace. What is "flood"? Is it some software that's using ttorrent? If so, that'd be a bug on their side, not ttorrent's. Please re-open if I'm mistaken.

**Comment by danmilon** [comment]
Oups, sorry! Yeah wrong project! 

On August 10, 2017 6:43:40 PM GMT+03:00, Maxime Petazzoni <notifications@github.com> wrote:
>Closed #208.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/mpetazzoni/ttorrent/issues/208#event-1201189822

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
